docs/guides/expectations/features_custom_expectations/how_to_add_example_cases_for_an_expectation.md

```diff
@@ -105,6 +104,22 @@ Sometimes you need to specify the precise type of the columns for each backend.
     "x": "INTEGER",
   },
 ```
+
+:::info
+While Pandas is fairly flexible in typing, Spark and many SQL dialects are much more strict. 
+
+You may find you wish to use data that is incompatible with a given backend, or write different individual tests for different backends. 
+To do this, you can use the `only_for` attribute, which accepts `pandas`, `spark`, `sqlite` or a SQL dialect:
```

Comment:
Love this! You may also want to mention that this can also take a list of backends.